rsl/eval.py

```diff
@@ -362,7 +362,8 @@ def accept_SelectAnyInstanceNode(self, node):
         
     def accept_SelectManyInstanceNode(self, node):
         where = self.accept(node.where)
-        value = self.runtime.select_many_from(node.key_letter, where)
+        order_by = self.accept(node.ordered_by)
```

Comment:
Inconsistent naming, order_by or ordered_by (I prefer order_by, but can't explain why)

Reply:
done. I will leave the RSL keyword "ordered_by" only because this matches MASL syntax